sopel/modules/admin.py

```diff
@@ -44,6 +44,27 @@ def setup(bot):
     bot.config.define_section('admin', AdminSection)
 
 
+class InvalidSection(Exception):
+    def __init__(self, section):
+        super(InvalidSection, self).__init__(self)
+        self.section = section
+
+    @property
```

Comment:
I was between this and overriding the `__str__` method to get a pretty message. I went with this to help differentiate this exception from most where a message is actually passed in as the first argument to the constructor (which is not done/desired here).

Reply:
Well, at least the message looks good. :wink: